frontend/views/components/Tooltip.vue

```diff
@@ -113,7 +113,7 @@ export default {
 <style lang="scss" scoped>
 @import "@assets/style/_variables.scss";
 
-.c-wrapper {
+.c-twrapper {
```

Comment:
It should not affect the rest because of the scope, I would keep the c-wrapper. If there is a generic css call c-wrapper, that's a mistake

Reply:
I should have explained this... actually renaming `c-wrapper` to `c-twrapper` or `c-gwrapper` (at groupPledgesGraph) has a reson, although I don't like it very much. Maybe @pieer you know a better solution. So, let me explain, by starting to rollback the class name to `c-wrapper` as before:

![vue-bug](https://user-images.githubusercontent.com/14869087/70040049-c6afe680-15b2-11ea-90d8-86a3eb40631b.png)

See those classes in the DOM `c-wrapper`, one for the graphic (blue outline) and other for the tooltip (red outline). For some reason, the `data-v-59ba5d0f` attribute is added also to the tooltip, so the tooltip will have not only its styles (orange styles) but also the styles applied to the graphic wrapper (green styles), breaking the layout.

After some experiments, I realized this only happens when applied to the component top-level element (the wrapper of everything inside that component).

This is really annoying because it made me realize that CSS in Vue isn't truly scoped. So for that reason using `c-wrapper` or `c-container` as the top class of every component can potentially lead to unexpected bugs as this one. So, I believe we should find a little more "unique" names, at least for those top-level elements. That's why I wrote `c-gwrapper` and `c-twrapper`.

Reply:
Wow, @sandrina-p ... is there a way you can add a shortened version of this explanation to the Style Guidelines CSS section? I'm still having some difficulty understanding it, but it's a bit clearer to me now. Maybe link to this comment as well.

Also, yeah, if @pieer you have any ideas on how to avoid this, please do let us know!